Add metadata JSONField to Incident (#790)
Uninett/Argus: changelog.d/+incident-metadata.added.md

Added an optional JSONField "metadata" to incident. This can be used for any

> additional info the glueservice would like to store on the incident that nees
> more structure than tags. The field have been added to the V2

Reviewer: ```suggestion
additional info the glue-service would like to store on the incident that needs
more structure than tags. The field has been added to the V2
```
